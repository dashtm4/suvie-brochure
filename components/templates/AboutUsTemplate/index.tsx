export { default } from './AboutUsTemplate'
export * from './AboutUsTemplate'
