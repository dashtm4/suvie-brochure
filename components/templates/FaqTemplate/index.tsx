export { default } from './FaqTemplate'
export * from './FaqTemplate'
