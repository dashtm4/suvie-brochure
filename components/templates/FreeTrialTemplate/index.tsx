export { default } from './FreeTrialTemplate'
export * from './FreeTrialTemplate'
