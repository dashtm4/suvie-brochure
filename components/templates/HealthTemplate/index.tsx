export { default } from './HealthTemplate'
export * from './HealthTemplate'
