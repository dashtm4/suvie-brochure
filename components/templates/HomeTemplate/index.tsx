export { default } from './HomeTemplate'
export * from './HomeTemplate'
