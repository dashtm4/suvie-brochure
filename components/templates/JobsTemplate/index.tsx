export { default } from './JobsTemplate'
export * from './JobsTemplate'
