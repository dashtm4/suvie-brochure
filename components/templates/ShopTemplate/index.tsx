export { default } from './ShopTemplate'
export * from './ShopTemplate'
