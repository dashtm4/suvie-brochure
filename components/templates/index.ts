export * from './PrivacyPolicyTemplate'
export * from './TermsAndConditionsTemplate'
export * from './FaqTemplate'
export * from './JobsTemplate'
export * from './HomeTemplate'
export * from './KitchenRobotTemplate'
export * from './ShopTemplate'
export * from './HealthTemplate'
export * from './SmartMealsTemplate'
export * from './RecipesTemplate'
export * from './ReviewsTemplate'
export * from './FreeTrialTemplate'
export * from './AboutUsTemplate'
