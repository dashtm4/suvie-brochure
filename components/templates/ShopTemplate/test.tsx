import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { ShopTemplate } from '@components/templates'

describe('<ShopTemplate />', () => {
  it('should renders correct <ShopTemplate />', () => {
    const wrapper = shallow(<ShopTemplate />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
