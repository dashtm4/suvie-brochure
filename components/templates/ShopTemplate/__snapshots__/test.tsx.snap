// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShopTemplate /> should renders correct <ShopTemplate /> 1`] = `
<main>
  <FeaturesBanner
    features={
      Array [
        Object {
          "image": Object {
            "height": 40,
            "url": "/images/affirm-logo.svg",
            "width": 40,
          },
          "title": "Pay as low as $33/ month",
        },
        Object {
          "image": Object {
            "height": 40,
            "url": "/images/100-day-home-trial-risk-free.svg",
            "width": 40,
          },
          "link": "/free-trial/",
          "title": "100-day home trial, risk-free",
        },
        Object {
          "image": Object {
            "height": 30,
            "url": "/images/logo-seal-black.svg",
            "width": 30,
          },
          "title": "1 Year Warranty",
        },
      ]
    }
  />
  <ShopFirstSection />
  <ShopSecondSection />
  <ShopThirdSection />
  <RobotTechnicalSpecs
    className="pt-20 bg-quaternary pb-11 md:pb-15"
  />
  <ShopFifthSection />
  <ShopSixthSection />
</main>
`;
